en/docs/reference/pricing.md

```diff
@@ -2,229 +2,145 @@
 
-## Plans
-
-|               | Free                   |  Pay As You Go    | Enterprise     |
-|---------------|------------------------|-------------------|----------------|
-| Cost Per Step | 5,000 steps per month *|$5 per 1,000 steps <br/> <ul><li>First 5,000 steps per month are free</li></ul>| Contact us for discounted pricing including: <br/><ul><li> Volume Discounts</li><li> Committed-use Discounts</li><li> Support Channels and Support SLA</li></ul>|
-
- \* Limited to a maximum of 3 components
-
-
-
-## Choreo pricing FAQ
-
-### How does Choreo pricing work?
-
-Choreo pricing is based on the number of [steps](#what-is-a-step) in your integration solution. Choreo offers three payment plans: Free, Pay As You Go, and Enterprise.
-
-#### Free 
-The Choreo free tier allows you to execute 5,000 [steps](#what-is-a-step) per month, limited to a maximum of 3 components. If you exceed the free tier, your application cannot make any additional requests to Choreo. 
-
-#### Pay As You Go (PAYG) 
-Pay As You Go is a flexible usage-based payment plan. While the first 5,000 steps are free per month, Choreo charges you $0.005 for each additional step based on your usage per month. For example, for 1,000 steps Choreo charges you $5. 
-
-Let’s assume you have used a total of 12,500 [steps](#what-is-a-step) for a particular month, your monthly usage for that month under the PAYG plan will be:
-
-    Chargeable steps usage: 12,500 - 5,000 = 7,500
-    The total charge for the usage: 7,500 * $0.005 ($5 for every 1,000 steps) = $37.50
-
-#### Enterprise 
-The Enterprise plan is suitable for large-scale usage of Choreo. You can access volume discounts and committed-use discounts with this plan. [Contact sales](https://wso2.com/contact/?ref=choreo) to get a discounted rate.
-
-### What is a step?
-A step (1 step) is a maximum of 500ms compute time on a container having 350MB of memory and 0.5 vCPU. Any additional compute time is counted as an additional step.  
-An incoming event or message to Choreo or an outgoing API call from Choreo is a minimum of 1 step.
+The pricing for your Choreo deployment is mainly determined based on the following:
 
+- **The number of components**
 
-## Pricing examples
+    You can have up to five free components ((i.e., any combination of component types), and pay for each additional component.
 
-### Example 1: REST API 
+- **Infrastructure cost**
 
-An organization has a flight booking system on Choreo. This system involves several APIs and databases. The scenario and the APIs being used in the system are described below.
+    The infrastructure cost consists of the charges to Choreo by the cloud platform for vCPU, RAM, storage, ingress, egress, and network. Choreo provides $100 worth of infrastructure free of charge to each organization.
```

Comment:
The infrastructure costing is not yet charged because the feature is not fully implemented. The infrastructure cost break down will be shown for PAYG and Enterprise customers on the billing portal and for free users we are yet to decide on how to show it. Therefore we don't need to include that information here as of now, and update the docs once the feature is completed, until then users are not charged with the Infra cost.

Reply:
Included the sentence "Currently, users on Pay As You Go and Enterprise plans can get their infrastructure costs from the cloud platform vendor."

Apparently, the Billing Portal will eventually provide a breakdown of the infrastructure costs, but this is not implemented yet.

Reply:
ack. @RukshiW  shall we remove this section in that case?

Reply:
Done

Reply:
Lets not remove, the document should be in tally with the web site, this information is already there on the site https://wso2.com/choreo/pricing/. Therefore lets have similar information here and we will update with more information once the feature is completed. The feature is planned to be release by late January or by the first week of February 2023

Reply:
Added it back.

Reply:
There are some points in the website that we have not added such as 

-Choreo infrastructure credits gives you approximately 1,000 vCPU hours to process your containerized workloads. Infrastructure costs include vCPU, RAM, storage, ingress and egress, and network charges.

- Infrastructure costs for each component (e.g., vCPU, RAM, network utilization - ingress/egress, and storage) come with zero markups. You can also use your own cloud provider and pay those costs directly.

And also add a note saying this is not yet charged and update when we do start charging. Else we will be misleading the users with a further documentation to pricing.